lesson03/util/util.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+)
+
+func FindAverage(arr []int) float64 {
+	if len(arr) == 0 {
+		fmt.Println("empty array")
+		return 1
```

Comment:
The average of an empty set should probably be 0, not 1. It is a good thing, that you decided to check for an empty, a good thing to always keep in mind, but in this case it is unnecessary. You can delete this whole if statement and get the correct result.